src/styles/components/_news.scss

```diff
@@ -1,76 +1,10 @@
 .controller-news.action-show {
-  #content > div.message {
+  #content div.news {
+    position: relative;
+    margin-bottom: 10px;
     border-radius: 3px;
     background-color: $color-history-bg;
     border: 1px solid $color-history-border;
     padding: $histry-padding;
   }
```

Comment:
|Redmine 6.0 with v1.1.0 bleuclair |trunk Redmine with this bleuclair|
|----|----|
|<img width="734" height="458" alt="screenshot 2025-08-06 11 11 44" src="https://github.com/user-attachments/assets/22017636-3531-4c63-b487-8bd649bdfc35" />|<img width="734" height="458" alt="screenshot 2025-08-06 11 12 17" src="https://github.com/user-attachments/assets/80f76e6b-ebd0-454c-b687-7dd4925b17d8" />|

Fixed the style definition for #content > div.message, enabling the news summary to be displayed with a white background.

Reply:
Additionally, in Bleuclair v1.1.0 the timeline line for comments was not displayed. However, after unifying the styles into _speech_balloon_layout.scss, the line is now shown correctly.
I believe this behavior is acceptable, as the missing timeline line in Bleuclair v1.1.0 was likely not intentional but rather an implementation oversight.